app/test.jsx

```diff
@@ -134,6 +137,16 @@ class App extends React.Component {
                             <option value="OUTSIDE">OUTSIDE</option>
                             <option value="">disable</option>
                         </select>
+                        Choose barrier bg color:&nbsp;
+                        <select defaultValue="" id="barrierBGColor" onChange={this.onColorChange}>
+                            <option value="GREEN">GREEN</option>
+                            <option value="RED">RED</option>
+                            <option value="YELLOW">YELLOW</option>
+                            <option value="ORANGERED">ORANGERED</option>
+                            <option value="PURPLE">PURPLE</option>
+                            <option value="BLUE">BLUE</option>
+                            <option value="DEEPPINK">DEEPPINK</option>
+                        </select>
```

Comment:
@brucebinary I just added some color here to test barrier color changed in realtime.

Reply:
Works pretty nicely. Nice job (: